components/exchange/SearchBar.js

```diff
@@ -79,15 +85,23 @@ const CharacterSelect = styled.div`
 `
 
 const CharButton = styled.button`
+  text-align: center;
   padding: 0 6px;
   background-color: ${({ theme }) => theme.colors.focusBackground};
   border: none;
+  border-radius: 5px;
   font-size: 16px;
   cursor: pointer;
 
   @media screen and (max-width: ${screenSizes.tablet.max}) {
     padding: 0 5px;
   }
+  :focus {
+    outline: none;
+    -webkit-box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
+    -moz-box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
+    box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
```

Comment:
This shadow is used a lot. It might be a good idea to place it in the theme. Like
```js
...
 shadows: { outline: '0px 0px 10px 2px #SOMETHING' }
...
```